acceptance/bundle/run/basic/script

```diff
@@ -0,0 +1,16 @@
+title "no run key specified"
+errcode trace $CLI bundle run
```

Comment:
nit: personally prefer smaller tests, easier to review/debug and faster to run. e.g. you could split error-raising commands from the rest.

if you have one command then you also don't need to specify title (it's test name), errcode and trace.

Reply:
Fair enough, but in this case it's only 5 commands so still small. The flip side is you end up with a bunch of really granular files / tests which does not seem optimal.